relay-server/server/nonK8s.go

```diff
@@ -0,0 +1,253 @@
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/google/uuid"
+	pb "github.com/kubearmor/KubeArmor/protobuf"
+	kl "github.com/kubearmor/kubearmor-relay-server/relay-server/common"
+	kg "github.com/kubearmor/kubearmor-relay-server/relay-server/log"
+)
+
+type PushLogService struct {
+}
+
+// HealthCheck Function
+func (ls *PushLogService) HealthCheck(ctx context.Context, nonce *pb.NonceMessage) (*pb.ReplyMessage, error) {
+	replyMessage := pb.ReplyMessage{Retval: nonce.Nonce}
+	return &replyMessage, nil
+}
+
+// addMsgStruct Function
+func addMsgStruct(uid string, conn chan *pb.Message, filter string) {
+	MsgLock.Lock()
+	defer MsgLock.Unlock()
+
+	msgStruct := MsgStruct{}
+	msgStruct.Filter = filter
+	msgStruct.Broadcast = conn
+
+	MsgStructs[uid] = msgStruct
+
+	kg.Printf("Added a new client (%s) for PushMessages", uid)
+}
+
+// removeMsgStruct Function
+func removeMsgStruct(uid string) {
+	MsgLock.Lock()
+	defer MsgLock.Unlock()
+
+	delete(MsgStructs, uid)
+
+	kg.Printf("Deleted the client (%s) for PushMessages", uid)
+}
+
+// PushMessages Function
+func (ls *PushLogService) PushMessages(svr pb.PushLogService_PushMessagesServer) error {
+	uid := uuid.Must(uuid.NewRandom()).String()
+	conn := make(chan *pb.Message, 1)
+	defer close(conn)
+	addMsgStruct(uid, conn, "all")
+	defer removeMsgStruct(uid)
+
+	var err error
+	for Running {
+		var res *pb.Message
+
+		select {
+		case <-svr.Context().Done():
+			replyMessage := pb.ReplyMessage{
+				Retval: 1,
+			}
+			svr.Send(&replyMessage)
+			kg.Printf("Closing Message stream for (%s)", uid)
+			return nil
+		default:
+			if res, err = svr.Recv(); err != nil {
+				kg.Warnf("Failed to recieve a message")
+				return err
+			}
+
+			msg := pb.Message{}
+
+			if err := kl.Clone(*res, &msg); err != nil {
+				kg.Warnf("Failed to clone a message (%v)", *res)
+				continue
+			}
+
+			tel, _ := json.Marshal(msg)
+			fmt.Printf("%s\n", string(tel))
```

Comment:
can we enable printing of this conditionally? Also no point in printing to stdout. We can use `kg.Printf`.